### ISSUE TITLE
processing images upload and validation

### ISSUE THREAD

**Issue by dreamsbond** [limoncello-php/app#36]
is there any place we would handle the images upload and validation via move_uploaded_file()?

**Comment by dreamsbond** [comment]
a standardize place, via WebRoute Controller?

**Comment by neomerx** [comment]
That's standard for PSR7. I think you should have a look at the spec [here](http://www.php-fig.org/psr/psr-7/#16-uploaded-files) and [here](http://www.php-fig.org/psr/psr-7/#36-psrhttpmessageuploadedfileinterface) for a method `moveTo`.

**Comment by dreamsbond** [comment]
thanks. so the processing upload file will still be better implemented in ApiRoute Controllers, right?

**Comment by neomerx** [comment]
Yes, you start handling the `$request` in a controller. Whether you fully handle it in the controller making it `fat` or split the logic with API level would be your application design choice.
